content/contribute/_index.md

```diff
@@ -263,7 +263,7 @@ justify: 'center'
 							</div>
 							<div id="other" style="display:none;">
 								<p>Jellyfin has several other sub-projects that use various languages. If any of these suit you, head over to the relevant project page and begin hacking.</p>
-								<p><b>Java:</b> The <a href="https://github.com/jellyfin/jellyfin-androidtv" style="color:#fff">Android TV</a> and <a href="https://github.com/jellyfin/jellyfin-android" style="color:#fff">Android</a> apps are written in Java; Android also includes the main WebUI.</p>
+								<p><b>Java/Kotlin:</b> The <a href="https://github.com/jellyfin/jellyfin-androidtv" style="color:#fff">Android TV</a> and <a href="https://github.com/jellyfin/jellyfin-android" style="color:#fff">Android</a> apps are written in Java and Kotlin; Android also includes the main WebUI.</p>
```

Comment:
Actually, the Web UI part is outdated as well, since the app doesn't bundle it anymore. Maybe this is better:
```suggestion
								<p><b>Kotlin/Java:</b> The <a href="https://github.com/jellyfin/jellyfin-androidtv" style="color:#fff">Android TV</a> and <a href="https://github.com/jellyfin/jellyfin-android" style="color:#fff">Android</a> apps are written in Kotlin and Java for some legacy code.</p>
```

Reply:
Thanks, I commited your change.